bw6/src/verifier.rs

```diff
@@ -40,7 +40,6 @@ impl Verifier {
         public_input: &AccountablePublicInput,
         proof: &SimpleProof,
     ) -> bool {
-        assert_eq!(public_input.bitmask.size(), self.pks_comm.keyset_size);
```

Comment:
That was a sanity check. We pad the actual public keys with points of unknown (by means of hash-to-curve) dlog, so the bitmask has to be padded with 0s.